kubernetes/hypertrace.sh

```diff
@@ -3,6 +3,34 @@
 set -eEu -o functrace
 script=$0
 
+function clean() {
+  EXIT_CODE=0
+  kubectl get ns ${HT_KUBE_NAMESPACE} ${KUBE_FLAGS} >/dev/null 2>&1 || EXIT_CODE=$?
```

Comment:
Maybe you can do "set +e" entering this, then revert it at the end

That way you use `$?` after running instead of manually doing EXIT_CODE

seems less cluttered..

Reply:
ohh! cool! will try this.